openwisp-utils-qa-checks

```diff
@@ -83,15 +83,18 @@ runisort() {
     || { echoerr "ERROR: Isort check failed!"; FAILURE=1; }
 }
 
+# This check is run when any of the following conditions is true:
+# 1. --force-checkcommit is specified
+# 2. This function is invoked from Travis building a Pull Request
 runcheckcommit() {
-  [ -n "$TRAVIS_PULL_REQUEST" ] || TRAVIS_PULL_REQUEST="false"
-  if [ "$SKIP_CHECKCOMMIT" = "false" -o "$TRAVIS_PULL_REQUEST" != "false" ]; then
+	if [[ $FORCE_CHECKCOMMIT = true ]] ||
+		 [[ $TRAVIS = true && $TRAVIS_PULL_REQUEST = true ]]; then
```

Comment:
what is `$TRAVIS = true`? Sounds like something that can break existing builds, please remove it.
We want to run this check only on pull requests for now.

Reply:
`TRAVIS = true` when the script is being invoked by Travis (https://docs.travis-ci.com/user/environment-variables/#default-environment-variables) but you are right. Removed.